cdap-docs/admin-manual/source/security/authorization.rst

```diff
@@ -56,6 +54,11 @@ individual extensions for configuring properties specific to that extension:
 
 - :ref:`Integrations: Apache Sentry <apache-sentry>`
 
+:ref:`Security extension properties <appendix-cdap-default-security>`, which are specified
+in ``cdap-site.xml``, begin with the prefix ``security.authorization.extension.config``.
+Please refer to the appendix on :ref:`cdap-site.xml <appendix-cdap-default-security>` for
```

Comment:
I think this sentence should be removed. We don't have extension properties in the cdap-default appendix. The description for these properties is in individual extensions. For Sentry, we've already described that on lines 52-55 above